relay-server/src/processing/logs/process.rs

```diff
@@ -166,6 +178,10 @@ fn normalize_log(log: &mut Annotated<OurLog>, meta: &RequestMeta) -> Result<()>
                 .to_string()
         });
 
+    log.attributes
+        .get_or_insert_with(Default::default)
+        .insert_if_missing("sentry.payload_size_bytes", || size_in_bytes);
```

Comment:
Should do this in `processing/logs/store.rs`, then it works correctly even with external Relays, where this will not (someone or an external Relay sets the attribute, then it never gets changed).